nmostesting/suites/IS1101Test.py

```diff
@@ -0,0 +1,3813 @@
+
+                if version == self.version[input_id]:
+                    return test.FAIL("Version should change")
+                valid, response = TestHelper.do_request(
+                    "GET", self.node_url + "senders/" + sender_id
+                )
+                if not valid:
+                    return test.FAIL(
+                        "Unexpected response from the Node API: {}".format(response)
+                    )
+                if response.status_code != 200:
+                    return test.FAIL(
+                        "The sender {} is not available in the Node API request: {}".format(
+                            sender_id, response.json()
+                        )
+                    )
+                try:
+                    version = response.json()["version"]
+                except json.JSONDecodeError:
+                    return test.FAIL("Non-JSON response returned from Node API")
+                except KeyError as e:
+                    return test.FAIL("Unable to find expected key: {}".format(e))
+
+                if version == self.version[input_id]:
+                    return test.FAIL("Version should change")
+
+                stable_count = 0
+                time_start = datetime.datetime.now()
+                while stable_count < 5:
+                    if datetime.datetime.now() > time_start + datetime.timedelta(
+                        seconds=15
+                    ):
+                        time.sleep(CONFIG.HTTP_TIMEOUT)
+                    valid, response = TestHelper.do_request(
+                        "GET", self.node_url + "senders/" + sender_id
+                    )
+                    if not valid:
+                        return test.FAIL(
+                            "Unexpected response from the Node API: {}".format(response)
+                        )
+                    if response.status_code != 200:
+                        return test.FAIL(
+                            "The sender {} is not available in the Node API request: {}".format(
+                                sender_id, response.json()
+                            )
+                        )
+                    try:
+                        version = response.json()["version"]
+                    except json.JSONDecodeError:
+                        return test.FAIL("Non-JSON response returned from Node API")
+                    except KeyError as e:
+                        return test.FAIL("Unable to find expected key: {}".format(e))
+                    if version != self.version[sender_id]:
+                        stable_count = 0
+                        self.version[sender_id] = version
+                    else:
+                        stable_count += 1
+
+                valid, response = TestHelper.do_request(
+                    "GET",
+                    self.compat_url + "senders/" + sender_id + "/status/",
+                    time.sleep(20),
+                )
```

Comment:
```suggestion
                valid, response = TestHelper.do_request(
                    "GET",
                    self.compat_url + "senders/" + sender_id + "/status/"
                )
```

Reply:
Thank you for [the fix](https://github.com/AMWA-TV/nmos-testing/pull/849)! It's merged so resolving this one.